packages/api/bin/backup.js

```diff
@@ -33,7 +33,7 @@ async function backup(table, region = 'us-east-1', folder = 'backups') {
   const dyno = Dyno(dynoParams);
   const stringify = new stream.Transform({ objectMode: true });
   stringify._transform = function _transform(record, enc, callback) {
-    const line = Dyno.serialize(record);
+    const line = JSON.stringify(record);
```

Comment:
serialization and JSON.stringify are very different I presume, so I assume testing demonstrated that it wasn't working with `Dyno.serialize`?

Reply:
It was not and did work with the stringify